src/main/resources/public/src/views/ProcessingStat.vue

```diff
@@ -0,0 +1,374 @@
+<template>
+    <div class="container">
+        <h2>Processing Statistics</h2>
+        <div class="form-group row">
```

Comment:
it's not a form

Reply:
Yep but I wanted to keep the exact form style and organization. Even if "form group" don't have a real impact on date type input visual it provides the good spacing between the different <div> tag. Is it a problem to use it ?